snapshotter.go

```diff
@@ -239,8 +235,8 @@ func (s *ShardSnapshotFinder) DownloadSnapshot(snapshot Snapshot) error {
 }
 
 func (s *ShardSnapshotter) ToWorkingCopy(snapshot Snapshot) (string, error) {
-	copy := (fmt.Sprintf("%s/working-%s-%s-%d", s.LocalPath, s.Stream, s.ShardId, time.Now().UnixNano()))
-	return copy, exec.Command("mv", snapshot.LocalFile, copy).Run()
+	copy := fmt.Sprintf("%s/working-%s-%s-%d", s.LocalPath, s.Stream, s.ShardId, time.Now().UnixNano())
+	return copy, os.Rename(snapshot.LocalFile, copy)
```

Comment:
Changed to os.Rename here and below, don't think there was any special reason to shell out to mv.